dbms/src/Interpreters/PredicateExpressionsOptimizer.cpp

```diff
@@ -34,15 +34,26 @@ namespace ErrorCodes
     extern const int UNKNOWN_ELEMENT_IN_AST;
 }
 
-static constexpr auto and_function_name = "and";
+namespace
+{
+
+constexpr auto and_function_name = "and";
```

Comment:
It's possible to get its name from Functions: NameAnd::name.

Reply:
No!

Code should not depend on `Functions/...` (only dependency on `FunctionsFactory` and `IFunction` is allowed).